clarifai.go

```diff
@@ -175,9 +175,9 @@ func getRequestBody(input string) (*bytes.Buffer, error) {
 }
 
 // Get image tag array from Clarifai's API response
-func getImageTagsArray(jsonStr string) ([]ClarifyTag, error) {
+func getImageTagsArray(jsonStr string) ([]ImageTag, error) {
```

Comment:
In Go having a method/variable start with a capital letter means this method/variable is exported - i.e. accessible from outside the package. If we were to move the clarifai stuff to it's own folder, we wouldn't be able to call `getImageTagsFromCLF`. Internal helper functions should start with a lower case (as you don't actually want them to be called outside the package). Also, I really think this should be a method called on the `ClarifaiClient` struct